language/move-lang/src/typing/core.rs

```diff
@@ -259,6 +259,23 @@ impl Context {
         self.is_current_module(m) && matches!(&self.current_function, Some(curf) if curf == f)
     }
 
+    pub fn is_in_script_context(&self) -> bool {
```

Comment:
nice! Does this need to be public though?

Reply:
Ah, it doesn't need to be public. I'll remove the `pub` and we can add it back if there is a use case.